Bump version for pan-os pack with polling wrapper script (#40583)
demisto/content: Packs/PAN-OS/ReleaseNotes/2_5_5.md

#### Scripts

##### New: PANOSQueryLogs

> - New: Added the PANOSQueryLogs polling wrapper script, which searches Palo Alto Networks firewall logs across several log types (threat, traffic, wildfire, URL, data, correlation, system, and decryption). It provides flexible filtering capabilities including IP addresses, time ranges, network zones, rules, ports, URLs, file hashes, and custom query strings, with configurable result limits up to 5,000 logs. Security teams can investigate network activity efficiently, analyze traffic patterns, and perform forensic analysis across their Panorama and Firewall infrastructure through automated log retrieval. This script is dependent on the Panorama integration and can be executed against either a Firewall device or a Panorama device, depending on the configured integration instance.

Reviewer: ```suggestion
- New: Added the PANOSQueryLogs script which wraps the **pan-os-query-logs** polling command. The command searches Palo Alto Networks firewall logs across several log types (threat, traffic, wildfire, URL, data, correlation, system, and decryption).
```

> - New: Added a new script- PANOSQueryLogs that is a polling wrapper script; This script searches Palo Alto Networks firewall logs across eight different log types (threat, traffic, wildfire, URL, data, correlation, system, and decryption). It provides flexible filtering capabilities including IP addresses, time ranges, network zones, rules, ports, URLs, file hashes, and custom query strings, with configurable result limits up to 5,000 logs. This enables security teams to efficiently investigate network activity, analyze traffic patterns, and perform forensic analysis across their Panorama and Firewall infrastructure through automated log retrieval. This script depends on the Panorama integration and can be executed against either a Firewall device or a Panorama device, depending on the configured integration instance.

Reviewer 2: ```suggestion
- New: Added the PANOSQueryLogs polling wrapper script, which searches Palo Alto Networks firewall logs across several log types (threat, traffic, wildfire, URL, data, correlation, system, and decryption). It provides flexible filtering capabilities including IP addresses, time ranges, network zones, rules, ports, URLs, file hashes, and custom query strings, with configurable result limits up to 5,000 logs. Security teams can investigate network activity efficiently, analyze traffic patterns, and perform forensic analysis across their Panorama and Firewall infrastructure through automated log retrieval. This script is dependent on the Panorama integration and can be executed against either a Firewall device or a Panorama device, depending on the configured integration instance.
```
